### PR TITLE
[MJAVADOC-624] tagletArtifacts throws exception in case artifact is a Java 9+ module

### PR DESCRIPTION
**Pull request by schelldorfer**

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/javadoc/JavadocUtil.java [hunk 1]
```diff
@@ -735,6 +735,12 @@ protected static boolean validateEncoding( String charsetName )
 
         for ( String s : classes )
```
Comment (elharo) [inline src/main/java/org/apache/maven/plugins/javadoc/JavadocUtil.java]: optional: s --> clazz

**Comment by elharo** [comment]
Tests failing:

[ERROR] Errors: 
[ERROR]   JavadocReportTest.testTagletArtifacts:1177 » MojoExecution An error has occurr...
[INFO] 
[ERROR] Tests run: 69, Failures: 0, Errors: 1, Skipped: 0
[INFO]

**Comment by rfscholte** [comment]
Closing this PR as it is missing an integration test to confirm the fix and prevent future regression.

**Comment by jira-importer** [comment]
Resolve #915

**Pull request closed without merging**